tests/test_files/sample/EGAformat.csv

```diff
@@ -0,0 +1,2 @@
+title,alias,description,subjectId,bioSampleId,caseOrControl,gender,organismPart,cellLine,region,phenotype
+test sample,test1,A test sample metadata CSV file based on the EGA submitter portal format,123456789abc,,,unknown,,,,some disease state
```

Comment:
we need more lines here, as in real world scientists would have multiple lines for this.

Reply:
Could you elaborate bit more? The way CSV is parsed atm is that each line of CSV creates one object so multiple lines would indicate multiple different samples in this case. I understand the need, but I'm wondering about implementation.

Reply:
yes, we our users will upload multiple samples in a csv, thus we need to be able to work with that.